### ISSUE TITLE
Relative reputation

### ISSUE THREAD

**Issue by winsvega** [Unirep/Unirep#147]
Have you considered that reputation is a relative function depending on the community?

**Comment by vimwitch** [comment]
Yes absolutely. Unirep is just a base layer that applications can build on top of. So for example, you could write an app where when a user votes they have to prove how much reputation they have, then their vote is weighted differently based on that.

e.g. User A and User B are both endorsing a post from User C. User A has 10 reputation and User B has 2 reputation. As a result the post from User C is endorsed with a total of 12 reputation.

This could be more complex too, like non-linear scaling of reputation so that if User A had 5000 reputation their endorsement might only be worth 20 rep, something like `y = 2ln(x)` which looks like this: 
<img width="369" alt="image" src="https://user-images.githubusercontent.com/631020/196223508-25aee8e0-7b9a-407c-90ee-e2f36b3c1838.png">

I might have misunderstood what you mean by relative, but most schemes can be implemented.

**Comment by winsvega** [comment]
Yes by relative I mean.

User 'a' has 20 rep by the community A, and -5 rep by the community B and so on.

How about this?

**Comment by vimwitch** [comment]
Yes! Each attester (smart contract) has its own reputation system. So a user has separate balances for every application/community. The user can make a proof from community A and give it to a user in community B, or a user could make a proof showing balances from community A and B.

**Comment by winsvega** [comment]
There would be nice to have a system where you can sign up to the communities you trust or persons and then see the rep status of the feed you get according to your communities. 

Because the problem in steemit or reddit is that a global total rep a post gets. And if they don't like your post then it gets banned. So users posts only whats popular.

When in reality maybe the Authentic  group would give -100 for reposts but if its still a good repost it worth ending up in your feed.
